src/features/user/RegisterTrees/RegisterTrees.tsx

```diff
@@ -1,8 +1,5 @@
-import DateFnsUtils from '@date-io/date-fns';
-import { MenuItem } from '@material-ui/core';
-import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
-import { MuiPickersOverrides } from '@material-ui/pickers/typings/overrides';
-import { ThemeProvider } from '@material-ui/styles';
+import { MenuItem, ThemeProvider } from '@mui/material';
```

Comment:
Menu item - slight difference in font size as compared to develop branch

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867877084

Reply:
Fixed in #1467